pulsar_metrics/metrics/utils.py

```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
 
             return percents
     except Exception as e:
-        print(f"Error while calculating population percentages: {str(e)}")
+        print(f"Error in get_population_percentages() while calculating population percentages: {str(e)}")
```

Comment:
Comment: if is_numeric_dtype(new) & is_numeric_dtype(reference): If this condition returns false. (generally return true or false) what is the execution work flow ?

Reply:
I see ! Something needs to be changed here got include categorical variables for both new and reference and a TypeError if not of the same type.